resources/views/components/chirps/list.blade.php

```diff
@@ -0,0 +1,11 @@
+<div {{$attributes->merge(['class'=>"mt-6 bg-white shadow-sm rounded-lg border rounded-lg"]) }}>
+    @foreach ($chirps as $chirp)
+        <x-chirps.chirp-card :chirp="$chirp"/>
+    @endforeach
+</div>
+<script>
+    document.addEventListener('DOMContentLoaded', function(){
+        renderComments();
```

Comment:
I would not do this, because you are calling methods here which reside elsewhere, and there are more than one of those methods so they all get called at the same time. it is better to put this domcontent loaded listener in each card. OR it is better to take the javascript out of the card and put it in a file for the page, which applies to all the code on the page

Reply:
if it's working here thats fine, but doing it this way might haunt you later